cmd/cmd.js

```diff
@@ -90,15 +91,15 @@ class Cmd {
               err.retry();
             } else {
               //slightly different assignment of name since it comes from child prompt
-              if (options.simple) {
+              if (options.simple || options.contractsOnly) {
                 embark.generateTemplate('simple', './', inputvalue);
               } else {
                 embark.generateTemplate('boilerplate', './', inputvalue, options.template);
               }
             }
           });
         }
-        if (options.simple) {
+        if (options.simple || options.contractsOnly) {
```

Comment:
At the top of action it could be:

`const constractOnly = options.simple || options.contractsOnly`

then all your if conditions are:
`if (contractsOnly)`

Reply:
Done! Thanks for the suggestion 👌